testing/termite/APKBUILD

```diff
@@ -0,0 +1,71 @@
+# Contributor: Ian Bashford <ianbashford@gmail.com>
+# Maintainer: Ian Bashford <ianbashford@gmail.com>
+pkgname=termite
+pkgver=15
+pkgrel=0
+_vtever=0.50.2-ng
+_util_commit=409b8449ab51fccf51057621168c9c15c54d4807
+pkgdesc="A simple VTE-based terminal"
+url="https://github.com/thestinger/termite/"
+arch="all"
+options="!check"  # No test suite.
+license="LGPL-2.1-or-later AND MIT"
+depends="$pkgname-terminfo"
+makedepends="intltool libtool automake autoconf gtk-doc gtk+3.0-dev
+	glib-dev pcre2-dev gnutls-dev gperf ncurses
+	pcre2 gnutls"
```

Comment:
don't think pcre2 and gnutls are required when their -dev verions are used

Reply:
you're right -- removed them.